h2o-py/h2o/h2o.py

```diff
@@ -967,6 +967,29 @@ def deep_copy(data, xid):
     return duplicate
 
 
+def models():
+    """
+    Retrieve the IDs all the Models.
+
+    :returns: Handles of all the models present in the cluster
```

Comment:
Flow also shows algo. I think the simple form of just ids is more useful though. Maybe in the future, we could add an option detailed=True and show more than that but for now, this makes the best sense to me.